packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -728,8 +729,17 @@ export class DataViewExpander {
                     const readAttributeResponseItem = responseItem as ReadAttributeAcceptResponseItemJSON;
 
                     const localAttributeResultForRead = await this.consumption.attributes.getAttribute({ id: readAttributeResponseItem.attributeId });
-                    const localAttributeDVOForRead = await this.expandLocalAttributeDTO(localAttributeResultForRead.value);
+                    const localAttributeForReadExists = localAttributeResultForRead.isSuccess;
```

Comment:
IMO this is not ok. isSuccess means anything went wrong here .. to go back to the old behavior please check the code and throw if it's another reason than notFound.

Same everywhere else.